packages/server/lib/db/migrations/20230326083713_create_oauth_session.cjs

```diff
@@ -0,0 +1,22 @@
+exports.up = function(knex, _) {
+    return knex.schema.withSchema('nango').createTable('_nango_oauth_sessions', function(table) {
+        table.uuid('id').notNullable();
+        table.string('providerConfigKey').notNullable();
```

Comment:
We use snake case for DB columns too. 

ps: this unfortunately means that the object fields are snake case too, which is not idiomatic in Typescript. But I haven't found a decent way to have DB columns in snake case and object fields in came case.

Reply:
We can use the snake_case for the DB column and still use the camel case for Object fields. In this case, it will just mean that we have to modify the result when we get it from the DB, and also when we persist. Thanks for noticing this

Reply:
@bastienbeurier I understand it is not a priority, but since I have the capacity I will use it in the DB and a camel in the object. I do not want us to incur debt especially when I have the capacity do challenge the problem.

Reply:
It looks like your message didn't print here for some reason. 

If you feel that it's not too complex to differentiate camel-case for model, snake-case for DB, then I'm happy if you go for it. 

At some point, we'll still have to refactor all the other models that are using snake case already.